generators/kubernetes/templates/db/couchbase.yml.ejs

```diff
@@ -16,7 +16,23 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
+<%_ if (kubernetesUseDynamicStorage) { _%>
+apiVersion: v1
```

Comment:
I think we should use KUBERNETES_CORE_API_VERSION

Reply:
done